dispatch/theme/event_widget.py

```diff
@@ -0,0 +1,37 @@
+from dispatch.theme.fields import CharField, TextField, ArticleField, ImageField, EventField, InvalidField
+from dispatch.theme import register
+from dispatch.theme.widgets import Zone, Widget
+from dispatch.apps.events.models import Event
+
+class Sidebar(Zone):
+    id = 'sidebar'
+    name = 'Sidebar'
+
+class UpcomingEventsWidget(Widget):
+    """Widget to display the next 5 widgets that are upcoming in the sidebar"""
+
+    id = 'upcoming-events'
+    name = 'Upcoming Events'
+    template = 'widgets/test-upcoming-events.html'
+
+    zones = [Sidebar]
+
+    title = CharField('Upcoming Events')
+    featured_event = EventField('Featured Event')
+
+    def context(self, data):
+
+        events = Event.objects.filter(is_published=True).order_by('start_time')[:5]
+        data['events'] = []
+
+        for event in events:
+
+            # Pull the top five events, and automatically ignore the featured event
+            if event.id != self.data['featured_event']:
+
+                data['events'].append({
+                    'title': event.title,
+                    'date': event.start_time
+                })
+
+        return data
```

Comment:
You can simplify this method:

```py
data['events'] = Event.objects.filter(is_published=True).exclude(pk=data['featured_event']).order_by('start_time')[:5]

return data
```

The event objects can be passed directly to the template -- no need to pick out only the fields you want. This keeps the code simpler and adds for more flexibility if users want to create a custom template.

I didn't think of this earlier, but you'll also want to consider how to handle past events. I think it makes sense to show events that are happening *up to the current day*. So if it's 8pm and an event started at 7pm we'd still show it, but we wouldn't show it the next day. It'd be good to add a test case for this as well.